contributor.md

```diff
@@ -1,10 +1,13 @@
 # All contributors ✨
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors-)
```

Comment:
@KeithanPhilander it shows here on line 4 and 6 that you touched something around this badge area. The all-contributors bot might start to give me issues when I want to add a new contributor. Can you create a new branch and make changes without touching anything between the start and end comments of the badge?